src/serialize.ts

```diff
@@ -28,7 +21,12 @@ const requireResolve =
  * Related: https://nextjs.org/docs/basic-features/data-fetching#reading-files-use-processcwd
  */
 function setEsbuildBinaryPath() {
-  const esbuildDir = pkgDir.sync(requireResolve('esbuild'))
+  /**
+   * Ensure we use node's native require.resolve method,
+   * webpack overrides require.resolve by default and returns the module ID
+   * instead of the resolved path
+   */
+  const esbuildDir = pkgDir.sync(eval('require.resolve')('esbuild'))
```

Comment:
Lurking with zero context here, is there any way to avoid using eval? It’d be awesome to be able to use this library inside edge functions, which don’t support eval.

Reply:
👋  I'll see what I can do!